pkg/worker/handler/container/handler.go

```diff
@@ -52,7 +52,7 @@ func New(c Config) *Handler {
 		gau[Metric] = recorder.NewGauge(recorder.GaugeConfig{
 			Des: "the health status of ecs service containers",
 			Lab: map[string][]string{
-				"service": {"alloy", "server", "specta", "worker"},
+				"service": {"alloy", "graphql", "otel-collector", "server", "specta", "worker"},
```

Comment:
Sneaking this in here to get rid of some warning logs we are currently seeing in our deployments on AWS.